### PR TITLE
Refactor/community naming cleanup

### PR DESCRIPTION
**Pull request by osmaczko**
- use `IsOwner` instead of `config.privateKey != nil`
- improve community functions naming

### CODE REVIEW

**Review by 0x-r4bbit — APPROVED** [review]
Nice cleanup, thank you

**Review by mprakhov — APPROVED** [review]
LGTM

File: protocol/communities/community.go [hunk 1]
```diff
@@ -1409,24 +1409,24 @@ func (o *Community) MarshaledDescription() ([]byte, error) {
...
+		}
 
-	// This should not happen, as we can only serialize on our side if we
-	// created the community
-	if o.config.PrivateKey == nil && len(o.config.MarshaledCommunityDescription) == 0 {
-		return nil, ErrNotAdmin
-	}
-
-	//We are not owner or admin, use the received serialized version
-	if !o.IsOwnerOrAdmin() {
```
Comment (mprakhov) [inline protocol/communities/community.go]: admins should also be available to marshal description

Comment (osmaczko) [reply protocol/communities/community.go]: Admins don't have private key so how can they sign the description?

Comment (mprakhov) [reply protocol/communities/community.go]: signing will be skipped, I think it is ok. At least if key is empty, we do not return an error

**Pull request merged**